doc/api/process.md

```diff
@@ -637,6 +637,48 @@ process's [`ChildProcess.disconnect()`][].
 If the Node.js process was not spawned with an IPC channel,
 `process.disconnect()` will be `undefined`.
 
+## process.dlopen(module, filename[, flags])
+<!-- YAML
+added: v0.1.16
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/12794
+    description: Added support for the `flags` argument.
+-->
+
+* `module` {Object}
+* `filename` {string}
+* `flags` {integer}.
+
+The `process.dlopen()` method allows to dynamically load shared
+objects. It is primarily used by `require()` to load
+C++ Addons, and should not be used directly, except in special
+cases. In other words, unless you have a good reason, you should
+prefer [`require()`][] over `process.dlopen` to load C++ Addons.
+
+The `flags` argument is an integer that allows to specify dlopen
+behavior. See the [`os.constants.dlopen`][] documentation for details.
+
+If for any reason you must use `process.dlopen` (for instance, because
+you need to specify dlopen flags), it's often useful to use
+[`require.resolve()`][] to lookup the module's path.
+
+One important drawback when calling `process.dlopen` is that you need
+to pass a `module` instance. Functions exported by the Addon,
+are acessible via `module.exports`.
```

Comment:
please avoid the use of informal pronouns like `you` in the docs. This could be reworded as:

```
*Note*: An important drawback when calling `process.dlopen()` is that the
`module` instance must be passed. Functions exported by the addon will
be accessible via `module.exports`.
```